storage/remote/queue_manager.go

```diff
@@ -367,6 +385,31 @@ func (t *QueueManager) Start() {
 	t.metrics.minNumShards.Set(float64(t.cfg.MinShards))
 	t.metrics.desiredNumShards.Set(float64(t.cfg.MinShards))
 
+	// Checking if the JSON file is corrupted
+	var check Checkpoints
+
+	jsonFile, err := os.Open("SegmentRecord.json")
+	defer jsonFile.Close()
+
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	err = json.Unmarshal(byteValue, &check)
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
We should use the queue managers logger rather than `fmt.Println`, you can look at usage in this file like `level.Info` or `level.Error` depending on what you want to log.